lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lc-form-validation",
-  "version": "1.0.0",
+  "version": "1.0.3",
```

Comment:
1.0.3 version was the last published

Reply:
We must upgrade to 1.1.0 after finish all branches?

Reply:
Agree, let's go for 1.1.0 once we have as well de formtofield thing